lib/google/amp/cache/configuration.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Google
+  module AMP
+    module Cache
+      class Configuration
+        attr_accessor :private_key, :amp_cache_domain
+
+        def initialize
+          @amp_cache_domain = 'cdn.ampproject.org'
+        end
+      end
+
+      # rubocop:disable ThreadSafety/InstanceVariableInClassMethod
```

Comment:
The configuration is not Thread-safe just as configurations of most Gems, couldn't figure out a better way to do so at this moment 🙇